### ISSUE TITLE
Show an initial first set of a repeating group

### ISSUE THREAD

**Issue by hlmr-erik** [open-formulieren/open-forms#2345]
### Thema / Theme

Frontend

### Omschrijving / Description

- When having a repeating group as required, at least one instance needs to be filled in. As it is now, a user still has to invoke the fields by clicking the "+" button. This seems unnecessary to me. Showing an initial set of fields in this case would fix this story https://github.com/open-formulieren/open-forms/issues/2344 as well; since the required-asterisk wouldn't be needed at all at the repeating-group level; the asterisks at the fields (now showing immediately on tab load) would suffice.

### Added value / Toegevoegde waarde

Make the initial appearance of fields match the intention set in the form.

**Comment by hlmr-erik** [comment]
Reading the documentation, it seems like a similar feature was planned/implemented before? https://open-forms.readthedocs.io/en/2.0.0_a/manual/forms/examples/repeating_groups.html 
![image](https://user-images.githubusercontent.com/111565013/201932422-05a854c6-7b21-4047-a1de-41aad904c1df.png)

However, I don't see this mentioned option in the properties window of 2.0.0:
![image](https://user-images.githubusercontent.com/111565013/201932614-d94ca2ea-95b5-4659-82f5-7b9b1ffeb58e.png)

**Comment by sergei-maertens** [comment]
Yes, but we ran into formio bugs and decided to scrap that option :(

**Comment by vjawala** [comment]
Is it possible to re-add this feature, as the formio bug is closed?

**Comment by sergei-maertens** [comment]
The upstream issue was closed because it was outdated, not because it was fixed. Additionally even if it was fixed, it's a mountain of work to upgrade to newer Formio versions since we had to patch a lot of things ourselves. Even upgrading to their latest 4.13.x patch release breaks Open Forms :(
